src/blocks/advanced-form/fields/radio/edit.js

```diff
@@ -392,7 +397,7 @@ function FieldRadio( { attributes, setAttributes, isSelected, clientId, context,
 					/>
 
 					{isSelected ?
-						<div className={'kb-form-multi'}>
+						<div className='kb-form-multi' >
```

Comment:
Is this change meaningful or just syntax preference?
Not important, just curious.

Reply:
Non of the above. I was just returning things to its initial state after I realized that was the wrong place to place the code.

Reply:
cool